### PR TITLE
add `task.common.focus_resource_in_explorer` based on new action `HIGHLIGHT_RESOURCE_IN_EXPLORER`

### PR DESCRIPTION
**Pull request by wangmingliang-ms**
What does this implement/fix? Explain your changes.
---------------------------------------------------
…

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-guidance/src/main/resources/flow-schema/tasks.json [hunk 1]
```diff
@@ -260,6 +260,29 @@
           }
         }
       }
+    },
+    {
+      "additionalProperties": false,
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-guidance/src/main/resources/flow-schema/tasks.json:265]: What's the usage of this parameter? Do we need to udpate the config/mapping model?

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/component/TreeUtils.java [hunk 2]
```diff
@@ -163,15 +171,49 @@ private static IntellijAzureActionManager.ActionGroupWrapper toIntellijActionGro
...
         renderer.append(Optional.ofNullable(view.getDescription()).map(d -> " " + d).orElse(""), SimpleTextAttributes.GRAY_ITALIC_ATTRIBUTES, true);
         renderer.setToolTipText(Optional.ofNullable(view.getTips()).orElse(view.getLabel()));
     }
+
+    public static boolean isInAppCentricView(@Nonnull DefaultMutableTreeNode node) {
+        return isInAppCentricView(new TreePath(node.getPath()));
+    }
+
+    public static boolean isInAppCentricView(@Nonnull TreePath path) {
+        if (path.getPathCount() < 2) {
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/component/TreeUtils.java:206]: Why path count < 2 is not app centric view? e.g., the root node of app centric (Resoruce Groups Node) ?

Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/component/TreeUtils.java:206]: root node is `empty`, Resource Groups node is actually not a root node.

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-guidance/src/main/resources/flow-schema/tasks.json (hunk 1 above)
Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-guidance/src/main/resources/flow-schema/tasks.json:265]: > What's the usage of this parameter?     

https://json-schema.org/understanding-json-schema/reference/object.html#additionalproperties

> Do we need to udpate the config/mapping model?    

no

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**